app/src/main/java/com/globant/example/mentorapp/home/domain/interactor/ListUsersInteractor.java

```diff
@@ -0,0 +1,68 @@
+package com.globant.example.mentorapp.home.domain.interactor;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import com.globant.example.mentorapp.home.domain.interactor.data.remote.APIService;
+import com.globant.example.mentorapp.home.domain.interactor.data.util.ApiUtils;
+import com.globant.example.mentorapp.home.domain.model.EventApiResponseEntity;
+import com.globant.example.mentorapp.home.domain.model.UserEntity;
+import com.squareup.otto.Bus;
+
+import android.support.annotation.NonNull;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+/**
+ * This class runs all web service users queries.
+ * Created by oscar.amezquita on 9/06/2017.
+ */
+
+public class ListUsersInteractor {
+
+    private APIService apiService;
+    private Bus bus;
+
+    @Inject
+    public ListUsersInteractor(APIService apiService, Bus bus) {
+        this.apiService = apiService;
+        this.bus = bus;
+    }
+
+    public void getUsersList() {
```

Comment:
Keep in mind that the interactor name must mean a Use Case, for example: FetchUserListInteractor. And it must contain a method name *execute()*